### ISSUE TITLE
[Wear OS] Completion Notification ask or [Suggestion]

### ISSUE THREAD

**Issue by tsukitaro** [Razeeman/Android-SimpleTimeTracker#218]
Good Morning

I hope this issue finds you well 

I want to ask you if is there some functionality to notify when a session is complete, I mean, when we start a session there is a option to set duration, but at the moment I only found this like void because the app not notify when the session is complete therefore I always go over the limit.
And although there is a feature to notify when a timer is running, this is set to certain duration, so if we define 30 min, the 10 min activities cannot be used with this feature.

**In summary I ask if there are any way to set notify or vibration in the wear OS when we complete sessions because now in the Samsung Watch 6 the timer over limit duration otherwise if there is not, I would like to suggest it**

**Comment by Razeeman** [comment]
Hello! Thank you for the feedback. Do you mean a notification when a goal time for the session is complete? It should be working if notifications for the app is enabled on phone and a system setting enabled to show notifications from the phone on the watch.

**Comment by tsukitaro** [comment]
Hello, thank you for your response 

Yes, I mean maybe it happened by the other notifications 

I will close this issue
But last question you know if the pomodoro feature will arrive to wear OS?

**Comment by Razeeman** [comment]
Pomodoro on the wear os is planned for future releases but can't tell for sure when it will be implemented.

**Comment by tsukitaro** [comment]
Thank you
